src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/freestyle/SecurityScanFreestyle.java

```diff
@@ -493,6 +543,26 @@ public void setCoverity_local(Boolean coverity_local) {
         this.coverity_local = coverity_local ? true : null;
     }
 
+    @DataBoundSetter
+    public void setCoverity_build_command(String coverity_build_command) {
+        this.coverity_build_command = coverity_build_command;
```

Comment:
We should use `Util.fixEmptyAndTrim()` method in case of String types. Look into other new setters methods too. I see that we are trimming off before putting these values into the parameters map (in the ScanParametersMap class). So we should trim it here.

Reply:
Done